com.endlessnetwork.MidnightmareTeddy.appdata.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.endlessnetwork.MidnightmareTeddy</id>
+  <name>Midnightmare Teddy</name>
+  <developer_name>Endless Network</developer_name>
+  <summary>Shoot and survive</summary>
+  <project_license>LicenseRef-proprietary</project_license>
+  <metadata_license>CC0-1.0</metadata_license>
+  <launchable type="desktop-id">com.endlessnetwork.MidNightmareTeddy.desktop</launchable>
+  <description>
+    <p>You're in a dream...or maybe a nightmare! All of the toys have come alive and are chasing you. Fight them off and run to survive for as long as you can. Want the ultimate challenge? Try out Math Mode and use your number skills as a weapon to fight back the evil toys!</p>
+  </description>
+  <releases>
+		<release version="1.0" date="2019-03-17"/>
+  </releases>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://github.com/endless-network/MidnightmareTeddy_Binary/raw/master/TeddySS1.png</image>
+	    <caption>Shoot all the enemies</caption>
+    </screenshot>
+    <screenshot>
+      <image>https://github.com/endless-network/MidnightmareTeddy_Binary/raw/master/TeddySS2.png</image>
+	    <caption>Solve problems to defeat enemies</caption>
+    </screenshot>
+    <screenshot>
+      <image>https://github.com/endless-network/MidnightmareTeddy_Binary/raw/master/TeddySS3.png</image>
+      <caption>Battle through waves of enemies</caption>
+    </screenshot>
+    <screenshot>
+      <image>https://github.com/endless-network/MidnightmareTeddy_Binary/raw/master/TeddySS4.png</image>
+      <caption>Survival or math mode</caption>
+    </screenshot>
+  </screenshots>
+  <content_rating type="oars-1.1">
+    <content_attribute id="violence-cartoon">moderate</content_attribute>
+    <content_attribute id="violence-fantasy">moderate</content_attribute>
+    <content_attribute id="violence-realistic">none</content_attribute>
+    <content_attribute id="violence-bloodshed">none</content_attribute>
```

Comment:
You can remove the none attributes with a none value.

Reply:
Didn't know that, thanks.  The huge pile of "none" value attributes have been removed.